java/statestore/src/main/java/sleeper/statestore/s3/S3FileInfo.java

```diff
@@ -96,37 +136,49 @@ public boolean equals(Object o) {
             return false;
         }
         S3FileInfo that = (S3FileInfo) o;
-        return Objects.equals(fileInfo, that.fileInfo) && status == that.status;
+        return externalReferences == that.externalReferences && Objects.equals(filename, that.filename) && Objects.equals(internalReferences, that.internalReferences);
```

Comment:
These `equals`,`hashCode`, and `toString` methods need regenerating to include `lastUpdateTime`

Reply:
Done, thanks :)